libs/payments/capability/src/lib/capability.manager.ts

```diff
@@ -2,9 +2,26 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import {
+  CapabilitiesResult,
+  ContentfulManager,
+  ServiceResult,
+} from '@fxa/shared/contentful';
 import { Injectable } from '@nestjs/common';
 
 @Injectable()
 export class CapabilityManager {
-  constructor() {}
+  constructor(private contentfulManager: ContentfulManager) {}
+
+  async getClients() {
+    const clients: ServiceResult[] = (
+      await this.contentfulManager.getServicesWithCapabilities()
+    ).serviceCollection.items;
```

Comment:
This mapping could be done with the util style that @bbangert has setup in https://github.com/mozilla/fxa/pull/15856/files#diff-b7c3ae916791042272450dbd27a3d4ec44ad2e7276a728829e4c323bfb0e20ddR26

Reply:
I'll take a look and shout if I have any questions, thanks!